content/millennium/dstu2/proprietary-codes.md

```diff
@@ -56,6 +56,10 @@ The DocumentReference Resource supports proprietary codes for:
 
 * DocumentReference.type codes are maintained in [Code Set 72 Event Code](#code-set-72-event-code)
 
+_Notes:_
+
+* When adding or updating data, if both proprietary and standard codes are populated for the specific request body field, the request will fail
```

Comment:
```suggestion
* On inbound requests (POST, PUT, PATCH), if both proprietary and standard codes are populated for the specific request body field, the request will fail
```

I'd accept either

Reply:
how will the request fail?

what http code will be returned?

what error message will be returned?

Reply:
this is not in a sensible spot on the page, it should not be in the middle of the resource sections

Reply:
any reason it's here for DSTU2 but in the bulleted list for R4?

Reply:
I suggest 'When adding or updating data, proprietary codes and standard codes may not be set in the same coding array. If they are, the create or update request will fail with a `422 Unprocessable Entity` response.'

could even link to client error 7 https://fhir.cerner.com/millennium/dstu2/#client-errors

Reply:
@MaxPhilips I had them in separate places because DocumentReference is the only DSTU2 resource affected and all are affected in R4. Where would a better place be?

Reply:
let's move it to the list of bullets up top

Reply:
[I combined both of your suggestions into this and moved it up](https://github.com/cerner/fhir.cerner.com/pull/313/files#diff-6fdcb00b5b1b6edd51b687794bea6b95R24)